Packs/CiscoISR/README.md

```diff
@@ -0,0 +1,52 @@
+# Cisco ISR
+This pack includes Cortex XSIAM content.
+
+## Configuration on Server Side
+You need to configure your Cisco ISR device to forward Syslog messages.
+
+Perform the following in order to configure log forwarding:
+1. Enter Global Configuration Mode by either typing **configure terminal** or **conf t**.
+2. Input the destination of where the logs should be sent by typing **logging \<Hostname OR IP Address\>**.
+
+The instructions above set the logging with default configuration values.
+
+More information can be found [here](https://www.cisco.com/c/en/us/td/docs/routers/sdwan/command/iosxe/qualified-cli-command-reference-guide/m-logging-commands.pdf)
+
+### Syslog Time Parsing Support
+Support for syslog timestamp parsing is available with UTC timezone only. You will need to add the year and milliseconds to the product default datetime format.
+* [Service timestamps command doc](https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/fundamentals/command/cf_command_ref/R_through_setup.html#wp4972384860)
+* [Clock Timezone command doc](https://www.cisco.com/c/en/us/td/docs/routers/xr12000/software/xr12k_r3-9/system_management/command/reference/yr39xr12k_chapter4.html#wp748744425)
+
+1. Enter Global Configuration Mode by either typing **configure terminal** or **conf t**.
+2. Configure your syslog timestamp formatting by typing the command;
+```bash
+service timestamps log datetime year msec
+```
+3. Optional, to configure the timezone format to UTC, type the command;
+```bash
+clock timezone UTC 0
+```
+4. Revert back to Privileged EXEC mode by typing **end**. 
+5. Save your changes by either typing 
+```bash
```

Comment:
Should this section be written as:
```bash
write memory
```
OR
```bash
copy running-config startup-config 
```

Reply:
yes